src/main/resources/reference.conf

```diff
@@ -46,4 +46,26 @@ videobridge {
     # Must be set when enabled = true
     # domain="some-domain"
   }
+  ice {
+    tcp {
+      # Whether ICE/TCP is enabled.
+      enabled = false
+      port = 443
+      # Port to fallback to in case binding to 'port' fails. Set to -1 to disable fallback.
+      fallback-port = 4443
```

Comment:
I'm curious: is there a reason you didn't end up using a list in the new config for the ports?

Reply:
Yes, because a list suggests that we're binding on multiple ports.

Reply:
That's really just a matter of how it's named/documented, no?

Reply:
Exactly. I think `port` and `fallback-port` serve as a good description of what they can do. It's immediately obvious how to change the port. With a list one has to read the documentation to figure out what it means and how to change the port.

I don't think we'll even need more than one fallback port, so what would be the reasons for exposing a list in the config file?

Reply:
Even for 2 ports, I think a list is more ergonomic as we just iterate through until one is successful.  It also avoids the problem of having to use `-1` if no fallback is desired and, if someone wanted to fallback to more ports they could do so easily.

Reply:
I don't think we should consider the case of more than two ports, there's no real use for it.

Do you know if the config library provides a way to read either an int or a list of ints? I'd be nice to just have `port = 443`, but allow it to be overriden by `port = [443, 4443]`

Reply:
I'd say the utility of multiple ports degrades with each one, but I don't think there's a cliff from useful -> not useful between 2 and 3, say.

As for the Int or list of Ints, I think we could do that but we'd have to do it on our own.  Basically we could have a fallback which first tried it as one type and then another, but we'd need to have a common 'final' type which would have to be a list of Ints.